### PR TITLE
Joy issue squashing

### PR DESCRIPTION
**Pull request by pm0kjp**

### CODE REVIEW

**Comment by pm0kjp** [comment]
I made changes to two files:  In the **R macros** I changed
> If you're stuck, reach out and ask for help! 

and replaced it with

> If you're stuck, reach out and ask for help! While the authors of these modules can't provide support for software installations, you may find assistance through [StackOverflow](https://stackoverflow.com/), a local user group (for instance, if your university or employer has an R User Group), or the IT department at your institution. Learning how to ask for help effectively is its own challenge: may we suggest our module [How to Troubleshoot](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/how_to_troubleshoot/how_to_troubleshoot.md#1) for tips?

In the **Intro to R, transform** I made a lot of updates to documenting the use of the double equals, boolean logic, and the pipe function.  Look especially at:

*  the bottom of the page at https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/joy-issue-squashing/r_basics_transform_data/r_basics_transform_data.md#12
* the top of the page at https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/joy-issue-squashing/r_basics_transform_data/r_basics_transform_data.md#13
* The entire page at https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/joy-issue-squashing/r_basics_transform_data/r_basics_transform_data.md#15
* The entire page at https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/joy-issue-squashing/r_basics_transform_data/r_basics_transform_data.md#16
* The entire page at https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/joy-issue-squashing/r_basics_transform_data/r_basics_transform_data.md#18

File: r_basics_transform_data/r_basics_transform_data.md [hunk 1]
```diff
@@ -253,13 +262,21 @@ The difference between `=` and `==`
 
 One common issue to be aware of is the difference between the single equals (=) and the double equals (==) operators.
 
-In R, using a single equals sign assigns a value.  It asserts "these things **are** equal."
+In R, using a single equals sign **assigns a value**.  It asserts "these things **are** equal."  
+
+* For example, `data = my_data` means "please set `data` equal to `my_data`."  
+
+The double equals sign does not assign, but **compares**.  It asks "**are** these things equal?".  That's why we use double equals in the context of a logical test that **compares** the left hand side, e.g. `mrn`, with the right hand side, e.g. 5000083, to check whether or not they are the same.
```
Comment (leemc-data-ed) [inline r_basics_transform_data/r_basics_transform_data.md:269]:
```suggestion
The double equals sign does not assign, but **compares**.  It **asks** "are these things equal?".  That's why we use double equals in the context of a logical test that **compares** the left hand side, e.g. `mrn`, with the right hand side, e.g. 5000083, to check whether or not they are the same.
```

File: r_basics_transform_data/r_basics_transform_data.md [hunk 2]
```diff
@@ -253,13 +262,21 @@ The difference between `=` and `==`
 
 One common issue to be aware of is the difference between the single equals (=) and the double equals (==) operators.
 
-In R, using a single equals sign assigns a value.  It asserts "these things **are** equal."
+In R, using a single equals sign **assigns a value**.  It asserts "these things **are** equal."
```
Comment (leemc-data-ed) [inline r_basics_transform_data/r_basics_transform_data.md:265]:
```suggestion
In R, using a single equals sign **assigns a value**.  It **asserts** "these things are equal."  
```

File: r_basics_transform_data/r_basics_transform_data.md (hunk 1 above)
Comment (leemc-data-ed) [reply r_basics_transform_data/r_basics_transform_data.md:269]: This is a little change, and I'm not a attached to it-- I just think that changing the words that are bolded here drives home the difference a little better. Feel free to leave it the way you had it if you disagree!

File: r_basics_transform_data/r_basics_transform_data.md [hunk 3]
```diff
@@ -362,24 +396,46 @@ D is not correct because it flips the positions of the comparison; the column na
...
-| OR (`|` in R) | True if at least one side is True | "Rabbits are reptiles OR rabbits are quadrupeds" is True |
-| OR (`|` in R) | False if and only if both sides are False | "Rabbits are reptiles OR rabbits are bipeds" is False |
-| NOT (`!` in R) | Turns True into False and False into True | ! "Rabbits are reptiles" is True, ! "Rabbits are mammals" is False |
 
-We also have to consider using parenthesis to ensure the proper order of operations.  The order of operations for boolean algebra, from highest to lowest priority is NOT, then AND, then OR.  Forgetting to account for order of operations is a common mistake by novice users of boolean logic.
+### Order of Operations
+
+We also have to consider using parenthesis to ensure the proper order of operations when we have more than two conditions we're considering.  Just like you need to think about parentheses in math, you might have to do the same thing with logical conditions!
+
+The order of operations for boolean algebra, from highest to lowest priority, is NOT, then AND, then OR.  Forgetting to account for order of operations is a common mistake by novice users of boolean logic.
```
Comment (leemc-data-ed) [inline r_basics_transform_data/r_basics_transform_data.md:433]:
```suggestion
The order of operations for boolean algebra, from highest to lowest priority, is **NOT**, then **AND**, then **OR**.  Forgetting to account for order of operations is a common mistake by novice users of boolean logic.
```

Comment (leemc-data-ed) [reply r_basics_transform_data/r_basics_transform_data.md:433]: Minor suggestion for emphasis.

**Review by leemc-data-ed — APPROVED** [review]
I have just a couple of minor suggestions, that honestly you could take or leave, so I'll go ahead and approve!

**Pull request merged**